workspace_tools/data_scrape/supported_html.py

```diff
@@ -0,0 +1,107 @@
+HEADER = """
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+
+table, th, td {
+    border: 1px solid black;
+     border-collapse: collapse;
+}
+
+.rotate {
+    height: 140px;
+    white-space: nowrap;
+    width: 30px;
+    position: relative;
+    top: 75px;
+     right: -50px;
+     -moz-transform: rotate(-45.0deg);  /* FF3.5+ */
+      -ms-transform: rotate(-45.0deg);  /* IE9+ */
+       -o-transform: rotate(-45.0deg);  /* Opera 10.5 */
+  -webkit-transform: rotate(-45.0deg);  /* Safari 3.1+, Chrome */
+          transform: rotate(-45.0deg);  /* Standard */
+}
+</style>
+</head>
+<body>
+<div id="users">
+<input class="search" placeholder="Search" />
+<table id="example" class="dataTable display" cellspacing="0" width="80%">
+"""
+
+AVAILABLE = ["NO", "YES"]
+SROW = "<tr>\n"
+EROW = "</tr>\n"
+SBODY = "       <tbody class=\"list\">\n"
+EBODY = "</tbody>\n"
+SHEAD = "<thead>\n"
+EHEAD = "</thead>\n"
+SHEADER1 = "<th><div class=\"rotate\" id=\"filter-"
+SHEADER2 = "\">"
+EHEADER = "</div></th>\n"
+SCELL1 = "          <td class = \""
+SCELL2 = "\">"
+ECELL = "</td>\n"
+
+FOOTER = """</table>
+</div>
+</body>
+"""
+
+SCRIPT1="""
+<script src="https://developer.mbed.org/media/uploads/Sissors/list.js"></script>
```

Comment:
One more comment. I do not think this is safe:
```html
<script src="https://developer.mbed.org/media/uploads/Sissors/list.js"></script>
```

With all do respect, who knows what kind of JavaScript is there in the future.
@0xc0170 We can't merge it with this kind of includes.